Formula/git-remote-hg.rb

```diff
@@ -0,0 +1,18 @@
+class GitRemoteHg < Formula
+  desc "Transparent bidirectional bridge between Git and Mercurial"
+  homepage "https://github.com/felipec/git-remote-hg"
+  url "https://github.com/felipec/git-remote-hg/archive/v0.3.tar.gz"
+  sha256 "2dc889b641d72f5a73c4c7d5df3b8ea788e75a7ce80f5884a7a8d2e099287dce"
+
+  depends_on :hg
+  depends_on :python if MacOS.version <= :snow_leopard
+
+  def install
+    inreplace "git-remote-hg", "python2", "python"
```

Comment:
I'd be surprised if they accept this to be honest. I believe on some Linux distros `#!/usr/bin/env python` points to `python3` now whereas `python2` is always "safe", so that's likely the reasoning.

OS X isn't particularly ahead of the curve. I tend to consider if a Apple bug rather than an upstream one.